src/config/proj_config.py

```diff
@@ -0,0 +1,19 @@
+import json
+import os
+
+
+class ProjConfig:
+    PROJ_ROOT_ABS_PATH = os.path.abspath('.')
+
+    @staticmethod
+    def getDeployment():
+        return ProjConfig.fromJSON(os.path.join(ProjConfig.PROJ_ROOT_ABS_PATH, 'config', 'files', 'deployment.json'))
```

Comment:
Ao rodar o pytest é retornado o erro:
`
============================= test session starts ==============================
platform linux -- Python 3.9.6, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
rootdir: /home/runner/work/back_fastAPI_template/back_fastAPI_template
plugins: cov-2.12.1
collected 0 items / 1 error

==================================== ERRORS ====================================
_____________________ ERROR collecting tests/test_main.py ______________________
tests/test_main.py:5: in <module>
    from src.main import ctrl
src/main.py:6: in <module>
    from src.init import Init
src/init.py:8: in <module>
    class Init:
src/init.py:11: in Init
    tipo_repo: REPOSITORIO = ProjConfig.getDeployment()[KEY.TIPO_REPOSITORIO.value],
src/config/proj_config.py:10: in getDeployment
    return ProjConfig.fromJSON(os.path.join(ProjConfig.PROJ_ROOT_ABS_PATH, 'config', 'files', 'deployment.json'))
src/config/proj_config.py:18: in fromJSON
    with open(path) as file:
E   FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/back_fastAPI_template/back_fastAPI_template/config/files/deployment.json'
=========================== short test summary info ============================
ERROR tests/test_main.py - FileNotFoundError: [Errno 2] No such file or direc...
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.44s ===============================
`
O certo nao seria (pergunta):
`return ProjConfig.fromJSON(os.path.join(ProjConfig.PROJ_ROOT_ABS_PATH, 'src', 'config', 'files', 'deployment.json'))`

O mesmo seria válido para o método `getFastapi`

Reply:
Arquivos de config foram duplicados num caminho dentro da pasta `./test/config/files`